adapter/service/chargepoint/service.go

```diff
@@ -85,8 +88,8 @@ type Service interface {
 	SendPhaseModeReport(force bool) (bool, error)
 	// SupportedStates returns states that are supported by the chargepoint.
 	SupportedStates() []string
-	// SupportsAdjustingCurrent returns true if the chargepoint supports adjusting current.
-	SupportsAdjustingCurrent() bool
+	// SupportsAdjustingMaxCurrent returns true if the chargepoint supports adjusting current.
+	SupportsAdjustingMaxCurrent() bool
```

Comment:
I think that it should check if supports offered current, and maybe in addition - the max current. @boguslaw-wojcik what do you think?

```suggestion
	// SupportsAdjustingOfferedCurrent returns true if the chargepoint supports adjusting offered current.
	SupportsAdjustingOfferedCurrent() bool
```